BeeSwift/CurrentUserManager.swift

```diff
@@ -105,11 +118,25 @@ class CurrentUserManager : NSObject {
         }
     }
     
+    func handleSuccessfulSignup(_ responseJSON: JSON) {
+        guard let authToken = responseJSON["authentication_token"].string, let username = responseJSON["username"].string else {
+            return
+        }
+        
+        CurrentUserManager.sharedManager.setApiToken(ApiToken(type: .AuthenticationToken, token: authToken))
+
+        UserDefaults.standard.set(username, forKey: usernameKey)
+        UserDefaults.standard.synchronize()
+        
+        NotificationCenter.default.post(name: Notification.Name(rawValue: CurrentUserManager.signedInNotificationName), object: self)
```

Comment:
After successfully signing UP, what about next signing IN with that data?

The sign in response contains data about default notifications.

Reply:
or fetching user, rather